app/controllers/providers/check_provider_answers_controller.rb

```diff
@@ -20,7 +20,7 @@ def continue
     private
 
     def set_variables
-      @proceeding_types = legal_aid_application.proceeding_types
+      @proceeding_type = legal_aid_application.lead_proceeding_type
```

Comment:
Should the instance variable also include the `lead_` prefix, as when this functionality extends it will have more significance in the view? So `@lead_proceeding_type`